expandFns.php

```diff
@@ -419,3 +419,15 @@ function equivalent_parameters($par) {
     default: return array($par);
   }
 }
+
+function str_almost_the_same($str1, $str2) {  // For comparing strings with forgiveness
```

Comment:
This is a slightly opaque name for the function; it's not quite clear how "almost" is "almost".  Would `str_equivalent` be clearer?